includes/free-trial/class-wc-calypso-bridge-free-trial-orders-notice.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/**
+ * Class WC_Calypso_Bridge_Free_Trial_Orders_Notice.
+ *
+ * @since   1.9.16
+ * @version 1.9.16
+ *
+ * Renders an admin notice on Orders page.
+ */
+class WC_Calypso_Bridge_Free_Trial_Orders_Notice  {
+	/**
+	 * The single instance of the class.
+	 *
+	 * @var object
+	 */
+	protected static $instance = null;
+
+	/**
+	 * Get class instance.
+	 *
+	 * @return object Instance.
+	 */
+	final public static function get_instance() {
+		if ( is_null( self::$instance ) ) {
+			self::$instance = new self();
+		}
+
+		return self::$instance;
+	}
+
+	public function __construct(){
+		if ( ! wc_calypso_bridge_is_ecommerce_trial_plan() ) {
+			return;
+		}
+
+		add_action('admin_notices', function() {
+			$screen = get_current_screen();
+			if ( 'edit-shop_order' === $screen->id ) {
+				?>
+				<div class="free-trial-orders-notice notice notice-info">
+                    <div>
+                        <h3><?php echo __('Start selling to everyone', 'wc-calypso-bridge')?></h3>
```

Comment:
Thanks for fixing the shorthand echos. Could you please add `;` at the end of these commands?

Reply:
🤦 Sorry about that. Updated in https://github.com/Automattic/wc-calypso-bridge/pull/936/commits/6fdd9fa02a76064ec8a65eca76b659a44a1d6bbb